### ISSUE TITLE
Can't get the same result as the pretrain model

### ISSUE THREAD

**Issue by YihuaZou** [thu-media/Comyco#9]
Thank you for your work！I have some problems.
1、I used cook_traces as the training set and cook_test_traces as the test set for training. The average QoE obtained in the end was 49+, while the average QoE obtained by the pretrain model was 51+. How can I train to get similar results to pretrain?
2、And the network structure in the code is different from the network structure described in the paper (for example, delay uses CNN in the code, but FC in the figure of the paper), which one should be used?
Look forward to your kind reply!

**Comment by godka** [comment]
Hi, Yihua,
Thanks for your interest.

1) How can I train to get similar results to pretrain?

Actually, I have no idea since I have run this code once before I submitted the pretrained model. 
Here are some points that may improve the performance: in

https://github.com/thu-media/Comyco/blob/0e8c1fc7dbda3cb7101e0b8f4f61489a1595f6c2/src/libcomyco.py#L38

You can use 

```
self.entropy = -tf.reduce_sum(tf.multiply(self.real_out, tf.log(self.real_out)), reduction_indices=1, keepdims=True)
...
self.loss -= 1e-3 * tf.reduce_sum(self.entropy)
```

2) the network structure in the code is different from the network structure described in the paper

Here's a minor typo in the paper. The delay information should be extracted by CONV-1D rather than FC.
BTW, welcome to consider different NN architecture, such as Transformer and AutoML.

Have a good day,
Tianchi
